tests/test_build_latex.py

```diff
@@ -480,12 +480,11 @@ def test_footnote(app, status, warning):
             '{\\phantomsection\\label{\\detokenize{footnote:bar}} '
             '\ncite\n}') in result
     assert '\\caption{Table caption \\sphinxfootnotemark[4]' in result
-    assert 'name \\sphinxfootnotemark[5]' in result
-    assert ('\\end{threeparttable}\n\\par\n\\endgroup\n%\n'
-            '\\begin{footnotetext}[4]\\sphinxAtStartFootnote\n'
-            'footnotes in table caption\n%\n\\end{footnotetext}%\n'
+    assert ('\\begin{footnotetext}[4]\\sphinxAtStartFootnote\n'
```

Comment:
thanks I will check but in fact now they are located inside table environment in first cell after header. This is needed so that footnotes from caption are not listed after footnotes from table cells. I will extend test to check correct positioning.